Build failing - missing curlbuild.h

Following the instructions, the build is failing due to missing curlbuild.h

```
/home/btcr/btcr-DID-method/src/curlWrapper.cpp:5:10: fatal error: curl/curlbuild.h: No such file or directory                                             
 #include <curl/curlbuild.h>                                                                                                                              
          ^~~~~~~~~~~~~~~~~~                                                                                                                              
compilation terminated.                                                                                                                                   
make[2]: *** [src/CMakeFiles/createBtcrDid.dir/build.make:128: src/CMakeFiles/createBtcrDid.dir/curlWrapper.cpp.o] Error 1                                
make[1]: *** [CMakeFiles/Makefile2:320: src/CMakeFiles/createBtcrDid.dir/all] Error 2                                                                     
make: *** [Makefile:141: all] Error 2
```

I have curl 7.65.0 installed and don't have a curlbuild.h in my /usr/include/curl directory.

If I removing the following line from curlwrapper.cpp, make completed without error:
```
#include <curl/curlbuild.h>
```

--- a/src/curlWrapper.cpp
+++ b/src/curlWrapper.cpp
@@ -2,7 +2,6 @@
 
 #include <curl/curl.h>
 #include <curl/easy.h>
-#include <curl/curlbuild.h>
 #include <sstream>
 #include <iostream>
 

